backend/src/main/java/hu/flowacademy/worksheet/entity/Worksheet.java

```diff
@@ -36,8 +38,8 @@
             })
     @Column(name = "worksheet_id", nullable = false)
     private String id;
-    @Column(name = "partner_id")
-    private String partnerId; // FIXME Change to many to one in the future.
+    @ManyToOne(fetch = FetchType.LAZY)
+    private Partner partnerId;
```

Comment:
Nevezd át nyugodtan partner-re. Csak ügylej rá, hogy a OneToMany-ben is frissítsd a mappedBy-t

Reply:
Kellett ott másra is ügyelni :D